include/inset_state.hpp

```diff
@@ -9,7 +9,7 @@
 #include "time_tracker.hpp"
 #include <boost/multi_array.hpp>
 #include <cairo/cairo.h>
-#include <nlohmann/json.hpp>
+#include "json.hpp"
```

Comment:
In the repository, the single-include header is in a subfolder `nlohmann`. We should follow the same format, so that our include statement becomes:
```
#include "nlohmann/json.hpp"
```